lib/network.js

```diff
@@ -1,22 +1,27 @@
 var assert = require('assert')
-var swarmDefaults = require('dat-swarm-defaults')
-var disc = require('discovery-swarm')
+var pump = require('pump')
+var hyperswarm = require('hyperswarm')
 
 module.exports = function (archive, opts, cb) {
   assert.ok(archive, 'dat-node: lib/network archive required')
   assert.ok(opts, 'dat-node: lib/network opts required')
 
   var DEFAULT_PORT = 3282
-  var swarmOpts = Object.assign({
-    hash: false,
-    stream: opts.stream
-  }, opts)
-  var swarm = disc(swarmDefaults(swarmOpts))
-  swarm.once('error', function () {
+  var swarm = hyperswarm()
+  swarm.listen(DEFAULT_PORT)
+  swarm.once('error', function (err) {
+    console.error(err)
```

Comment:
We didn't log errors before, why now?

Reply:
yeah good point, just testing. i will remove before merge